pom.xml

```diff
@@ -788,5 +807,48 @@ and
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>download-war</id>
+      <activation>
+        <file>
+          <missing>${env.JENKINS_WAR}</missing>
+        </file>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>jenkins-war</id>
+                <goals>
+                  <goal>copy</goal>
+                </goals>
+                <phase>process-test-resources</phase>
+                <configuration>
+                  <artifactItems>
+                    <artifactItem>
+                      <groupId>org.jenkins-ci.main</groupId>
+                      <artifactId>jenkins-war</artifactId>
+                      <version>${jenkins.version}</version>
+                      <type>war</type>
+                    </artifactItem>
+                  </artifactItems>
+                  <outputDirectory>${project.build.directory}</outputDirectory>
+                  <stripVersion>true</stripVersion>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>lts</id>
+      <properties>
+        <jenkins.version>2.387.1</jenkins.version>
```

Comment:
This will need to be updated by the release lead after shipping an LTS release, just like in BOM.

Reply:
Filed https://github.com/jenkinsci/acceptance-test-harness/issues/1098 for later improvement